operator/image.yaml

```diff
@@ -66,10 +66,7 @@ packages:
   remove:
     - vim-minimal
     - subscription-manager
-    - shadow-utils
-    - tar
     - libuser
-    - findutils
```

Comment:
I think `tar` and `findutils` are needed in the Kafka images only. So does findutils, nmap-ncat etc.

In general ...
* https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/base/Dockerfile#L12 is what we install in the base image shared for operators and Kafka
* https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/kafka-based/kafka/Dockerfile#L12 is what we add for Kafka images
* We do not install any additional packages for operators (https://github.com/strimzi/strimzi-kafka-operator/blob/main/docker-images/operator/Dockerfile)

Reply:
Thanks @scholzj